document/core/text/instructions.rst

```diff
@@ -214,6 +214,92 @@ Lexically, an |Toffset| or |Talign| phrase is considered a single :ref:`keyword
    \end{array}
 
 
+.. index:: atomic memory instruction
+   pair: text format; instruction
+.. _text-instr-atomic-memory:
+
+Atomic Memory Instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _text-atomic-load:
+.. _text-atomic-loadn:
+.. _text-atomic-store:
+.. _text-atomic-storen:
+.. _text-atomic-rmw:
+.. _text-atomic-rmwn:
+
+The offset immediate to atomic memory instructions is optional, and defaults to
+:math:`\T{0}`.
+
+.. math::
+   \begin{array}{llclll}
+   \production{instruction} & \Tplaininstr_I &::=& \dots \\ &&|&
+     \text{i32.atomic{.}load}~~m{:}\Tmemarg_4 &\Rightarrow& \I32.\ATOMICLOAD~m \\ &&|&
```

Comment:
Ah. Yeah, I've seen that ugliness, too, and I don't understand why Mathjax does it. Seems like a bug to me, it doesn't happen with proper Latex (and the \mathsf environment isn't representing text). Good to know that the braces fix it, I might want to do that elsewhere as well then